server/src/api/community.rs

```diff
@@ -495,9 +501,19 @@ impl Perform for Oper<FollowCommunity> {
         };
       }
     } else {
-      // TODO: still have to implement unfollow
       let user = User_::read(&conn, user_id)?;
-      user.send_follow(&community.actor_id, &conn)?;
+
+      if data.follow {
+        // Dont actually add to the community followers here, because you need
+        // to wait for the accept
+        user.send_follow(&community.actor_id, &conn)?;
+      } else {
+        user.send_unfollow(&community.actor_id, &conn)?;
+        match CommunityFollower::ignore(&conn, &community_follower_form) {
```

Comment:
Shouldnt this be `unfollow()`? Ignore is confusing.

Reply:
Agree, this is an easy enough change since its in the trait.